src/patternfly/components/Tabs/tabs.scss

```diff
@@ -733,3 +781,35 @@ $pf-v6-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl
   }
 }
 // stylelint-enable
+
+@media (prefers-reduced-motion: no-preference) {
+  .#{$tabs}:not(.pf-m-box) {
+    .#{$tabs}__link,
+    .#{$tabs}__item.pf-m-action {
+      &::after {
+        content: revert;
+      }
+    }
+
+    .#{$tabs}__list::after {
+      position: absolute;
+      inset-block-start: var(--#{$tabs}--link-accent--InsetBlockStart, var(--#{$tabs}--link-accent--start));
+      inset-block-end: var(--#{$tabs}--link-accent--InsetBlockEnd);
+      inset-inline-start: var(--#{$tabs}--link-accent--InsetInlineStart, var(--#{$tabs}--link-accent--start));
+      width: var(--#{$tabs}--link-accent--Width, var(--#{$tabs}--link-accent--length));
+      height: var(--#{$tabs}--link-accent--Height, var(--#{$tabs}--link-accent--length));
+      content: "";
+      border: 0 solid var(--#{$tabs}--link-accent--color);
+      border-block-end-width: var(--#{$tabs}--link-accent--BorderBlockEndWidth);
+      border-inline-start-width: var(--#{$tabs}--link-accent--BorderInlineStartWidth);
+      transition:
+        --#{$tabs}--link-accent--length var(--#{$tabs}--link-accent--TransitionDuration--length) var(--#{$tabs}--link-accent--TransitionTimingFunction--length),
+        --#{$tabs}--link-accent--start var(--#{$tabs}--link-accent--TransitionDuration--start) var(--#{$tabs}--link-accent--TransitionTimingFunction--start),
```

Comment:
```suggestion
        --#{$tabs}--link-accent--start var(--#{$tabs}--link-accent--TransitionDuration--start) var(--#{$tabs}--link-accent--TransitionTimingFunction--start)
```

Reply:
I'm wondering about breaking this up rather than using the shorthand. It is a little harder to make sure the timings are lined up - but also, do we need to have separate timing function and duration variables, or could we have one for the entire effect - that we'd always want them to be the same since it's one motion.